book/P2C9_calib.qmd

```diff
@@ -100,7 +100,7 @@ $$
 
 where $O_1,...,O_n$ is the observed number of events  in $n$ groups and $E_1,...,E_n$ is the expected number of events.
 
-To utilise this test, the $[0,1]$ codomain of $S_i$ is cut into $B$ disjoint contiguous intervals ('bins') over the full range $[0,1]$.
+To utilize this test, the $[0,1]$ codomain of $S_i$ is cut into $B$ disjoint contiguous intervals ('bins') over the full range $[0,1]$.
 Let $m$ be the total number of observations, then assuming a discrete uniform distribution as the theoretical distribution, the expected number of events in each bin is $E_i = m/B$ (as the probability of an observation falling into each bin is equal).
 
 The observations in the $i$th bin, $b_i$, are defined by the set:
```

Comment:
Sets in mathcal notation?
observed should be lower case O_i -> o_i?
The overall notation confuses me here, i=1,...,n are the subjects?, better g=1,\ldots,G
Should it be [S_i(t_j)B] \in (b_{g-1}, b_{g}], where b_{g} is the border of the g-th interval?
\hat{S}_i is now the surv prob estimate of the i-th group? but we only have subject specific \hat{S} estimates?